x-pack/legacy/plugins/apm/public/components/shared/TransactionActionMenu/TransactionActionMenu.tsx

```diff
@@ -107,7 +107,7 @@ export const TransactionActionMenu: FunctionComponent<Props> = (
         { defaultMessage: 'Show trace logs' }
       ),
       condition: true,
-      hash: `/link-to/logs`,
+      path: `/link-to/logs`,
```

Comment:
Great! Can you also add a type definition like @weltenwort suggested? Just so this doesn't happen again :)